Add staking on fearless and intro page for participating (#59)
sora-xor/sora-docs: src/participate.md

# Participate in SORA Economy

> In this section you will find guidance on how to:

Reviewer: ```suggestion
In this section you will find all the information you need to:
```
